openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -556,28 +556,45 @@
 - control_key: IA-5 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - key: a
       text: |-
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-327
+        The certificates used for in-cluster mTLS authentication
+        are issued by an internal CA, at the moment it is not possible
```

Comment:
nit: we might not want to say 'at the moment' here, it's hard to imagine it being supported anytime soon so not to give that impression :P

Reply:
ha, removed